<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NepomukPage</class>
 <widget class="QWidget" name="NepomukPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="kcfg_nepomukEnabled">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="whatsThis">
      <string>Check this to enable semantic tagging of the downloaded images, using Nepomuk.</string>
     </property>
     <property name="text">
      <string>Enable Nepomuk tagging</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KEditListBox" name="kcfg_tagBlacklist">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="whatsThis">
      <string>Insert tags that you do not want to appear in your searches, regardless of the rating. They will also be excluded if the Nepomuk tagging is enabled.</string>
     </property>
     <property name="title">
      <string>Blacklisted tags</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="buttons">
      <set>KEditListBox::Add|KEditListBox::Remove</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
